### ISSUE TITLE
For xvsy, markers are still being plotted when gm.data.priority = 0

### ISSUE THREAD

**Issue by zshaheen** [UV-CDAT/vcs#214]

**Comment by doutriaux1** [comment]
should probably read:
`template.data.priority=0`

**Comment by jypeter** [comment]
This reminds me of something I have noticed this week in 2.8, but I did not have time to investigate, because I was helping a colleague with a script. Don't have time now, so just describing below what I noticed

I got the impression that even though the script explicitly set **tpl.comment1.priority = 0**, the comment1 was plotted when the script used **x.plot(tpl, v, comment1="Some comment")**. I thought that maybe this was not a bug, but a feature, like: if you are requesting comment1 in x.plot, override the tpl.comment1.priority

Don't have time to reproduce this, so if anybody has some feedback on this... Thanks!

**Comment by doutriaux1** [comment]
@jypeter yes I think you're right if you pass a keyword we turn `priority` back on. But I think you're right it might not be desirable.

**Comment by zshaheen** [comment]
@doutriaux1 This is still an issue. Here's how to recreate it:

1. Follow steps 1-6 to create an env here: https://acme-climate.github.io/acme_diags/docs/html/install-config-run.html#environment-for-development
2. Open the created html page and navigate to 'Zonal mean line plots' -> 'ANN'. This is the image
3. To edit the vcs script that created this file, look at this file: https://github.com/ACME-Climate/acme_diags/blob/master/acme_diags/plot/vcs/zonal_mean_xy_plot.py . Specificlly, it's line 231 what causes the error, setting it to 0 makes it dotted.
